docs/Makefile

```diff
@@ -20,9 +20,51 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) sou
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
 
-.PHONY: help clean watch html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext
+.PHONY: help clean watch html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest coverage gettext as_setup .EXPORT_ALL_VARIABLES
+
+# This sets up Autosubmit for Sphinx. It writes experiments to the build directory,
+# inside the sub-directory autosubmit. Note, this is a phony target, and uses
+# `.EXPORT_ALL_VARIABLES`, which was also made phony. All variables defined here
+# are exported to other targets. Use this feature with care!
+as_setup: .EXPORT_ALL_VARIABLES
+	$(eval AUTOSUBMIT_DIRECTORY := $(shell realpath -m $(BUILDDIR)/autosubmit))
```

Comment:
Something like `$(shell realpath '.')/$(BUILDDIR)/autosubmit` might work? You can test locally...